### ISSUE TITLE
Fix Travis CI

### ISSUE THREAD

**Issue by clarissalimab** [falko-org/Falko#360]
## Description
Travis build seems not to be working even though the environment works properly locally.

### Expected behavior
The CI build should be passing.

## Checklist
- [x] This issue has a significant name.
- [x] This issue has labels.
- [ ] This issue has screenshots (if necessary).

**Comment by MatheusRich** [comment]
@clarissalimab If your team could, this should be a priority, since we must assure that the code on every PR is working.

**Comment by alaxalves** [comment]
@clarissalimab @MatheusRich I agree

**Comment by alaxalves** [comment]
Hey guys, I really appreciate the very good work you're providing to Falko. But if could prioritize this issue in the next sprint it would be great. Thnx in advance @clarissalimab @brunapinos @wallacybraz @danielmarques28 @FilipeKN4

**Comment by clarissalimab** [comment]
Hi @alaxalves, this issue is in progress already. Sorry for the delay, we are trying to figure out a problem with karma.

**Comment by clarissalimab** [comment]
@alaxalves I thought the problem was the e2e command, but I found out that the problem has something to do with PhantomJS browser. I keep receiving the following warnings in travis:
```
15 05 2019 02:55:38.197:WARN [launcher]: PhantomJS have not captured in 60000 ms, killing.
15 05 2019 02:55:40.201:WARN [launcher]: PhantomJS was not killed in 2000 ms, sending SIGKILL.
15 05 2019 02:55:42.204:WARN [launcher]: PhantomJS was not killed by SIGKILL in 2000 ms, continuing.
```

It seems that Travis (or even other CIs tools) really have an issue with PhantomJS. I read that changing to Chrome Headless could work, so I will try tomorrow.

**Comment by alaxalves** [comment]
> @alaxalves I thought the problem was the e2e command, but I found out that the problem has something to do with PhantomJS browser. I keep receiving the following warnings in travis:
> 
> ```
> 15 05 2019 02:55:38.197:WARN [launcher]: PhantomJS have not captured in 60000 ms, killing.
> 15 05 2019 02:55:40.201:WARN [launcher]: PhantomJS was not killed in 2000 ms, sending SIGKILL.
> 15 05 2019 02:55:42.204:WARN [launcher]: PhantomJS was not killed by SIGKILL in 2000 ms, continuing.
> ```
> 
> It seems that Travis (or even other CIs tools) really have an issue with PhantomJS. I read that changing to Chrome Headless could work, so I will try tomorrow.

Thank you for taking time into this @clarissalimab. PhantomJS is really a pain in the ass when it comes to CI tools. Maybe headless chrome will do the trick. \*\*fingers crossed\*\*

**Comment by clarissalimab** [comment]
@alaxalves Done! I was able to change from PhantomJS to ChromeHeadless and now Travis can start the browser correctly!

Buuuut, I saw now that 27 unit tests weren't passing, so @brunapinos went to check out why they weren't passing and unfortunately, some of the tests passing happened because the lines that were supposed to verify the test's statements were commented. haha :crying_cat_face: We need to fix the tests.

What do we do now? We can remove the unit testing script from travis to work on fixing them on the following sprints (as the tests are not working anyways, but at least cypress tests seem to be working just fine) or we continue working on them in this issue?

We are running Travis on our fork, if you want to check our logs: https://travis-ci.org/gces-falko/Falko

**Comment by clarissalimab** [comment]
*((sad song plays in background))*
![image](https://user-images.githubusercontent.com/15739610/57858118-dd355280-77c6-11e9-9341-4ef6cdc7b453.png)

**Comment by ThalissonMelo** [comment]
I think that the best option is really remove the unit test from script and create the issue to add back when the tests are fixed.

**Comment by alaxalves** [comment]
@clarissalimab I agree

> I think that the best option is really remove the unit test from script and create the issue to add back when the tests are fixed.

**Comment by alaxalves** [comment]
Fixed in #368
